packages/ai/src/prompt/wrap-gateway-error.ts

```diff
@@ -12,9 +12,7 @@ export function wrapGatewayError(error: unknown): unknown {
     return new AISDKError({
       name: 'GatewayError',
       message:
-        'Vercel AI Gateway access failed. ' +
-        'If you want to use AI SDK providers directly, use the providers, e.g. @ai-sdk/openai, ' +
-        'or register a different global default provider.',
+        'Missing AI Gateway Key. Get one here: https://vercel.link/4rcaGME',
```

Comment:
```suggestion
        'Unauthenticated. Configure AI_GATEWAY_API_KEY or configure and use a provider module. Learn more: https://ai-sdk.dev/docs/getting-started/choosing-a-provider#ai-gateway'',
```

page is not in production yet, we can ship the change once it is